### PR TITLE
Fixed NullReferenceException in OpenXmlElement RawOuterXml field

### PR DESCRIPTION
**Pull request by aidilumarov**
The current version of your code:

It sets _rawOuterXml to string.Empty if value is null, but instead of leaving the setter immediately, it resets the field to null at the end of the setter

The changes I made should resolve the issue.

Thanks.

### CODE REVIEW

**Comment by twsouthwick** [comment]
@aidilumarov Thanks for the catch! Can you add a regression test so that it will be caught in the future?

**Comment by twsouthwick** [comment]
@aidilumarov I'd like to get a 2.12.0 release out this week. If you can get some tests in, we'll have this as part of that release.

Also, can you update the CHANGELOG.md to include this?

**Comment by aidilumarov** [comment]
@twsouthwick Sorry for taking long. I was off for several weeks. I have made the changes you asked for. Kind regards.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## Version 2.12.1 - 2020-12-19
+
+### Fixed
+- Fixed a `System.NullReferenceException` when setting _rawOuterXml field in `DocumentFormat.OpenXml.OpenXml` (#818)
+- Added regression tests in `DocumentFormat.OpenXml.Tests.OpenXmlElementTest2` to check for the mentioned exceptions in the future (#818)
```
Comment (twsouthwick) [inline CHANGELOG.md:11]: This line is unneeded. I try to keep the changelog focused on observable changes to the library

File: src/DocumentFormat.OpenXml/Properties/launchSettings.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "profiles": {
+    "DocumentFormat.OpenXml": {
```
Comment (twsouthwick) [inline src/DocumentFormat.OpenXml/Properties/launchSettings.json:3]: This file and the other one appear unrelated to the change. I believe this is for vs code. If so, happy to take it but would like the rest of the projects supported as well. Best in a separate pr though

**Review by twsouthwick — CHANGES_REQUESTED** [review]
Looks like the build is broken. We have warnings as errors on CI builds

**Comment by twsouthwick** [comment]
I've taken the liberty of updating the branch so it can be merged

**Review by twsouthwick — APPROVED** [review]

**Pull request merged**